bootfiles/bootinst.sh

```diff
@@ -40,5 +40,13 @@ if [ "$DEV" != "$PART" ]; then
    ) | fdisk $DEV >/dev/null 2>&1
 fi
 
+# Now to be able to boot from most of UEFI devices
+EFIDIR="$BOOT/EFI/boot"
+mkdir -p "$EFIDIR"
+cp efi-files/* "$EFIDIR"
+
+# I didn't rename the binary in place so it could be updated more easily
+mv "$EFIDIR/syslinux.efi" "$EFIDIR/bootx64.efi"
+
```

Comment:
I would like to suggest to copy the syslinux.cfg file to the EFI directory as a starting point:
```
cp syslinux.cfg "$EFIDIR"
```